service/telemetry/telemetry.go

```diff
@@ -59,7 +76,6 @@ func newLogger(cfg LogsConfig, options []zap.Option) (*zap.Logger, error) {
 	zapCfg := &zap.Config{
 		Level:             zap.NewAtomicLevelAt(cfg.Level),
 		Development:       cfg.Development,
-		Sampling:          toSamplingConfig(cfg.Sampling),
```

Comment:
In this current form, this is a breaking change that drops existing configuration functionality. 

`LogsSamplingConfig` was applied to all the logs. Now, we are changing it to be applied to exporter logs only. There are other places where the logs can be pretty noisy when triggered by data handling errors. This config option was initially introduced to resolve that specific issue in the batch processor: https://github.com/open-telemetry/opentelemetry-collector/issues/4554. And now, we are dropping that functionality.

Until we pass the sampledLogger to all the possibly noisy outputs, we cannot apply this change. But I don't think it's necessary at all.

I suggest we just introduce a reasonable default to the existing sampling configuration and remove the additional hardcoded sampling in the exporterhelper. By default, we can probably sample 100 logs in a second after 10 of the initial logs. That should be enough not to miss important logs and ensure it's applied to the noisy ones only. This will simplify the code and will not introduce any additional configuration options or exposed Go API.

@open-telemetry/collector-approvers WDYT?

Reply:
Hi @dmitryax ,
I agree we should not break customer already using `Sampling` in the general logger. I kept this configuration.

Setting a default configuration for the extra sampled logger.

For future work, we should use this sampled logger for noisy outputs. 

Thank you.

Reply:
Discussed the suggested approach during the Collector SIG meeting. No objections to going ahead with it. We need to make sure it's documented with instructions on how to disable the sampling